2-SMACSS-practice/katherine-smith-2/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Choco Pizza</title>
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/style.css">
```

Comment:
With the SMACSS method you no longer need a styles.css page. You take the content and separate them between the base, module, and layout.  I'm removing 2 points. You can resubmit for full credit.